### DISCUSSION TITLE
Welcome to bookworm Discussions!

### DISCUSSION THREAD

**Discussion by mush42** [mush42/bookworm#60] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Cambridgeport90** [comment]
I have been a user of the Bookworm software seemingly as soon as it came out on the scene, and I love it no less today than I did when it was new. I look forward to seeing new developments.

From: Musharraf ***@***.***>
Sent: Monday, May 3, 2021 9:21 AM
To: blindpandas/bookworm ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [blindpandas/bookworm] Welcome to bookworm Discussions! (#60)

👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

  *   Ask questions you’re wondering about.
  *   Share ideas.
  *   Engage with other community members.
  *   Welcome others and are open-minded. Remember that this is a community we
build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by k-kolev1985** [comment]
I'm Kostadin Kolev, from Bulgaria. I'm a translator and beta tester of software for Windows and Android. I like reading e-books. I've started doing that since 2004, when the first bulgarian TTS engine for Windows was created. Back then reading of e-books was done in MS Word or Notepad. Now I use programs like Bookworm and I'm very happy that they exist, because they make book reading much easier and more pleasant.

**Comment by Cambridgeport90** [comment]
That they do.

**Comment by cary-rowen** [comment]
Hello! When will Bookworm's next update be? Thanks!

**Comment by cary-rowen** [comment]
Read epub It would be nice to be able to use web style parsing.

**Comment by mush42** [comment]
> Hello! When will Bookworm's next update be? Thanks!

 Hello,
We're putting the final touches to make the next release the best release of Bookworm yet.

**Comment by eirinn1975** [comment]
Hello,
I've tried to install Bookworm 64bit, seems like Windows is considering it virus infected. Has anybody else experienced that?

**Reply by mush42** [reply]
Hi @eirinn1975 

This is a known issue. Microsoft Defender Antivirus flags any PyInstaller executable as a virus.

Unfortunately we can do nothing about this. All we're doing is to wait for Microsoft Defender Antivirus  to update their database.

In the mean time, you can make an exception for Bookworm executable.

Regards,

**Comment by Cambridgeport90** [comment]
You have to tell defender Smart Screen to simply ignore it. It was a pain for me too, at first.

**Comment by cary-rowen** [comment]
Hello @mush42 

I am a Simplified Chinese translator of Bookworm, I found that you use the word "document" most of the time, and the word "book" in a few cases, can you explain the difference between them?

thanks for your efforts

**Reply by mush42** [reply]
It is a personal preference. I think document is more generic than book.
That is, every book is a document, but the opposite is not true.
Ideally, every reference to book should be replaced by document.

**Reply by cary-rowen** [reply]
Yes, I agree with that.
